### DISCUSSION TITLE
Is it possible to customize the metrics while using the agent?

### DISCUSSION THREAD

**Discussion by tuhao1020** [open-telemetry/opentelemetry-java-instrumentation#6229] [Q&A]
Is it possible to customize the metrics while using the agent?

**Comment by trask** [comment]
hi @tuhao1020! what specifically do you want to customize?

**Comment by tuhao1020** [comment]
I want to add some labels to the metrics

**Reply by trask** [reply]
check out discussion going on here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/6209

**Reply by tuhao1020** [reply]
thanks!

**Comment by jkwatson** [comment]
I'm going to transfer this over to the instrumentation repo
